include/alpaka/acc/Tag.hpp

```diff
@@ -56,3 +57,32 @@ namespace alpaka
     template<typename TAcc, typename... TTag>
     inline constexpr bool accMatchesTags = (std::is_same_v<alpaka::AccToTag<TAcc>, TTag> || ...);
 } // namespace alpaka
+
+// include all Acc's because of the struct AccIsEnabled
+// if an acc is not include, it will be not enabled independent of the compiler flags
+// include after defining the traits alpaka::AccToTag and alpaka::TagToAcc because it
+// is a circular dependency
+#include "alpaka/acc/AccCpuOmp2Blocks.hpp"
+#include "alpaka/acc/AccCpuOmp2Threads.hpp"
+#include "alpaka/acc/AccCpuSerial.hpp"
+#include "alpaka/acc/AccCpuSycl.hpp"
+#include "alpaka/acc/AccCpuTbbBlocks.hpp"
+#include "alpaka/acc/AccCpuThreads.hpp"
+#include "alpaka/acc/AccFpgaSyclIntel.hpp"
+#include "alpaka/acc/AccGpuCudaRt.hpp"
+#include "alpaka/acc/AccGpuHipRt.hpp"
+
+namespace alpaka
+{
+    /// @brief check if the accelerator is enabled for a given tag
+    /// @tparam TTag alpaka tag type
+    template<typename TTag, typename = void>
+    struct AccIsEnabled : std::false_type
+    {
+    };
+
+    template<typename TTag>
+    struct AccIsEnabled<TTag, std::void_t<TagToAcc<TTag, alpaka::DimInt<1>, int>>> : std::true_type
+    {
+    };
+} // namespace alpaka
```

Comment:
Would it be safer to move this to a separate header file, _e.g._ `AccIsEnabled.h`, that can include all the others with out risk of a circular dependency ?

Reply:
Your are right. I fixed it.